main/guides/getting-started/sell-concert-tickets-contract-explainer.md

```diff
@@ -15,17 +22,39 @@ const inventory = {
   middleRow: {
     tradePrice: AmountMath.make(istBrand, 2n),
     maxTickets: 5n,
+  },
+  backRow: {
+    tradePrice: AmountMath.make(istBrand, n),
+    maxTickets: n,
   }
 };
 ```
+Our contract uses this `inventory` object as a parameter to initiate. 
+After the contract is initialized, a new [ERTP mint](https://docs.agoric.com/glossary/#mint) for the "Ticket" asset is created.
+
+<details>
+<summary>Note: AssetKind expresses type of assets</summary>
+
+There are three types of [assets](https://docs.agoric.com/guides/ertp/#asset). You can determine the [type of your asset](https://docs.agoric.com/reference/ertp-api/ertp-data-types.html#assetkind) by referring to the provided documentation.
+
+In our example, tickets are non-fungible and can have duplicates, meaning there can be many tickets of a single type. So, we are using `AssetKind.COPY_BAG`.
+
+</details>
 
-After the contract is initialized, a new ERTP mint for the "Ticket" asset is created:
 ```js
 const ticketMint = await zcf.makeZCFMint('Ticket', AssetKind.COPY_BAG);
 const { brand: ticketBrand } = ticketMint.getIssuerRecord();
 ```
 
-The entire inventory of tickets is minted and held by the `inventorySeat`:
+Once our asset is defined, we will mint our inventory at the start of our the smart contract and allocate it to our `inventorySeat`, which will function as our vendor.
```

Comment:
*which will function as our vendor* seems a bit misleading because a seat nothing like a vendor. Maybe just say something like: *allocate it to our `inventorySeat` object.*